test/verify/check-menu

```diff
@@ -44,6 +45,32 @@ class TestMenu(MachineCase):
         b.click('#about button[data-dismiss="modal"]')
         b.wait_popdown("about")
 
+        # Test session timeout
+        time.sleep(10)
+        with b.wait_timeout(1):
+            b.wait_not_visible("#session-timeout-dialog")
+            b.wait_visible("#filter-menus")
+            b.enter_page("/system")
+            b.mouse("#system_information_hardware_text", "mousemove", 24, 24)
+            b.switch_to_top()
+            time.sleep(34)
+            b.wait_visible("#session-timeout-dialog")
+            if int(b.text("#session-timeout-dialog #timeout-message").split()[-2]) < 20:
+                raise Exception("Unexpected remaining time in session")
+            b.click("#keep-session-alive")
+            time.sleep(34)
+            if int(b.text("#session-timeout-dialog #timeout-message").split()[-2]) < 20:
+                raise Exception("Unexpected remaining time in session")
+        time.sleep(30)
+        b.wait_visible("#login")
+        b.wait_visible("#login-info-message")
+        b.wait_text("#login-info-message", "You have been logged out due to inactivity.")
+        m.execute("sed -i 's/IdleTimeout = 1/IdleTimeout = 0/' /etc/cockpit/cockpit.conf")
+        m.execute("systemctl stop cockpit.service")
+        b.reload()
+        self.login_and_go("/system")
+        time.sleep(65)
```

Comment:
Sure, the next tests implicity make sure that the session still works, but for better readability, or when we decide to move this into its own test case, please add some assertions here. Like that the timeout dialog is not visible, and e. g. that a click on the user menu works.